pkg/rtree/check.go

```diff
@@ -0,0 +1,31 @@
+package rtree
+
+import (
+	"encoding/hex"
+
+	"github.com/google/btree"
+	"github.com/pingcap/log"
+	"go.uber.org/zap"
+)
+
+// CheckDupFiles checks if there are any files are duplicated.
+func CheckDupFiles(rangeTree *RangeTree) {
```

Comment:
IMO this function is irrelevant to RangeTree and should be placed in its original package. Not a big deal anyway.